src/components/MainPage.js

```diff
@@ -0,0 +1,16 @@
+import { DesktopBreakpoint, MobileBreakpoint } from './Breakpoint'
+import DesktopLayout from './desktop/DesktopLayout'
+import MobileLayout from './mobile/MobileLayout'
+
+const MainPage = () => (
+  <>
+    <DesktopBreakpoint>
+      <DesktopLayout />
+    </DesktopBreakpoint>
+    <MobileBreakpoint>
+      <MobileLayout />
+    </MobileBreakpoint>
+  </>
```

Comment:
Thoughts on breaking up this child-rendering logic (from `Breakpoint.js`) into a chain of conditions? The current approach is functional, but the markup itself might be confusing since it does not imply only one layout is active at a time.

e.g. something like the following instead:
```javascript
import { useIsDesktop } from 'somewhere'

const MainPage = () => {
  const isDesktop = useIsDesktop()
  return isDesktop ? <DesktopLayout /> : <MobileLayout />
}
```

Reply:
Great idea, that seems more clear to me.